bigtable_automation/gcf/manifest.go

```diff
@@ -0,0 +1,104 @@
+package gcf
+
+import (
+	pb "github.com/datacommonsorg/tools/bigtable_automation/gcf/proto"
+	"google.golang.org/protobuf/proto"
+)
+
+type ImportParams struct {
```

Comment:
Why not using the protobuf struct directly but have go struct here and do conversions?